packages/lib-classifier/src/plugins/tasks/DrawingTask/components/tools/DrawingToolRoot.js

```diff
@@ -1,46 +1,107 @@
 import PropTypes from 'prop-types'
 import React, { useState } from 'react'
 
-const STROKE_WIDTH = 1.5
-const SELECTED_STROKE_WIDTH = 2.5
+const STROKE_WIDTH = 3
+const SELECTED_STROKE_WIDTH = 5
 
-function DrawingToolRoot ({ children, tool }) {
-  const [ active, setActive ] = useState(true)
+function DrawingToolRoot ({ children, mark, svg, tool, onDelete }) {
+  const drawingTool = React.createRef()
+  const [ active, setActive ] = useState(false)
+  const [ eventCoords, setEventCoords ] = useState({ x: null, y: null })
   const mainStyle = {
     color: tool && tool.color ? tool.color : 'green',
     fill: 'transparent',
     stroke: tool && tool.color ? tool.color : 'green',
-    strokeWidth: active ? SELECTED_STROKE_WIDTH : STROKE_WIDTH
+  }
+
+  function convertEvent (event) {
+    const type = event.type
+
+    const svgEventOffset = getEventOffset(event.clientX, event.clientY)
```

Comment:
Client coords change when the window is scrolled. Use page coordinates or lock the window scroll position while drawing.

Reply:
I think this will only be a problem if the page scrolls while you're in the middle of a drawing interaction.